src/vstest.console/InProcessVsTestConsoleWrapper.cs

```diff
@@ -81,7 +80,12 @@ public InProcessVsTestConsoleWrapper(ConsoleParameters consoleParameters)
         }
 
         // Fill the parameters.
-        consoleParameters.ParentProcessId = Process.GetCurrentProcess().Id;
+        consoleParameters.ParentProcessId =
+#if NET7_0_OR_GREATER
```

Comment:
I should have called this earlier but can you please add a helper for this so that the pragma is only in one place and all calling sites have a simpler code?

Reply:
This change requires a lot of additional changes, (aka consolidating our fill-ins, making some internals visible etc. - we can do this, but in a different pr.)